config.h.cmake

```diff
@@ -27,4 +27,8 @@
 
 #cmakedefine HAVE_LIBCURL
 
+#define BUILD_DATA_DIR "${BUILD_DATA_DIR}"
+
+#define BUILD_CONFIG_DATA_DIR "${BUILD_CONFIG_DATA_DIR}"
+
```

Comment:
I can make them relative to the build directory (probably), but yes.  This
allows us to support out of tree builds.  For run-time usage, we still fall
back on the install path.

On Tue, Jul 12, 2016 at 12:17 PM, M. Teufel <notifications@github.com>
wrote:

> In config.h.cmake
> <https://github.com/SuperTux/supertux/pull/481#discussion_r70480607>:
>
> > @@ -27,4 +27,8 @@
> >
> >  #cmakedefine HAVE_LIBCURL
> >
> > +#define BUILD_DATA_DIR "${BUILD_DATA_DIR}"
> > +
> > +#define BUILD_CONFIG_DATA_DIR "${BUILD_CONFIG_DATA_DIR}"
> > +
>
> Do we need to hardcode these?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/SuperTux/supertux/pull/481/files/757132a36c1a17b3dbebc63ed4bd9322351f9703#r70480607>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AC7N4WNLPgpBKEAHDdLrpb9BFXLkpBqXks5qU8wmgaJpZM4JI07H>
> .
>